great-idea/css/services.css

```diff
@@ -0,0 +1,225 @@
+footer, header, hgroup, menu, nav, section {
+  display: block;
+}
+body {
+  line-height: 1;
+}
+ol, ul {
+  list-style: none;
+}
+blockquote, q {
+  quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+  content: '';
+  content: none;
+}
+table {
+  border-collapse: collapse;
+  border-spacing: 0;
+}
+
+/* Set every element's box-sizing to border-box */
+* {
+  box-sizing: border-box;
+}
+
+
+/* with html, body I get bottom content stuck in middle  */
+/*        of screen */
+body {
+  height: 100%;
+  font-family: 'Titillium Web', sans-serif;
+  display: flex;
+  flex-direction: column;
+  width: 880px;
+  margin:  0 auto;
+
+}
+
+h1, h2, h3, h4, h5 {
+  font-family: 'Bangers', cursive;
+  letter-spacing: 1px;
+  margin-bottom: 15px;
+}
+
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+.nav-container {
+  height: 40px;
+  width: 880px;
+  position: fixed;
+  z-index: 1;
+  background: darkblue;
+  top: 0;
+  display: flex;
+  justify-content: space-between;
+}
+
+
+.navbar_top {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+a{
+  padding-right: 50px;
+}
+
+.navbar_top a{
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  color: #BABABA;
+  text-decoration: none;
+  font-family: 'Raleway', sans-serif;
+  font-weight: bolder;
+}
+
+.navbar_top a:hover {
+  font-size: 16px;
+  color: white;
+}
+
+.navbar_top a:last-child {
+  padding-left: 56px;
+}
+
+.logoIMG {
+  width: 191.78px;
+  height: 40px;
+  background: lightgrey;
+  text-align: center;
+}
+
+
+h1 {
+  font-size: 70px;
+  width: 250px;
+  font-family: 'Questrial', sans-serif;
+}
+
+button {
+  font-size: 26px;
+  border: 1px solid black;
+  width: 220px;
+  border-radius: 5px;
+  margin-left: 18px;
+}
+
+button:hover {
+  background: lightgray;
+}
+
+.services-header-img {
+  display: flex;
+  flex-direction: column;
+  width: 880px;
+  padding: 80px 0 10px 0;
+}
+
+
+
+.main {
+  border-bottom: 2px solid black;
+  padding: 20px 0 40px 0;
+}
+
+.services-banner h1 {
+  font-size: 40px;
+  display: flex;
+  flex-direction: column;
+  height: 40px;
+}
+
+h2 {
+  font-family: 'Black Han Sans', sans-serif;
+  font-size: 20px;
+}
+
+.all-services {
+  display: flex;
+  align-items: flex-start;
+  flex-wrap: wrap;
+ /* height: 100%; */
+  justify-content: space-between;
+  padding-bottom: 35px;
+  border-bottom: 2px solid black;
+}
+
+.service {
+  border: 1px solid black;
+  padding: 25px 15px 5px 15px;
+  width: 420px;
+  margin: 40px 0 0 0;
+  background: #F4F3F4;
+}
+
+.service button {
+  margin: 20px 0;
+}
+
+.service-footer {
+  display: flex;
+  justify-content: space-between;
+  height: 260px;
+  margin-top: 40px;
+}
+
+.facts-services {
+  display: flex;
+  flex-direction: column;
+  width: 410px;
+  height: 250px;
+  align-self: flex-start;
+}
+
+.facts-services ul {
+  padding: 15px 0;
+  list-style-type: square;
+  padding-left: 40px;
```

Comment:
list alignment can be pretty annoying. https://www.w3schools.com/cssref/pr_list-style-position.asp that will show you how to align your bullets without guessing and checking on the padding.